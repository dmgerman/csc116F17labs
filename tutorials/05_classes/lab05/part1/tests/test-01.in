10
30
1
16