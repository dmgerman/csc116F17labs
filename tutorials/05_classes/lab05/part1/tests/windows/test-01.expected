Enter initial hour: 
Enter initial minute: 
c.get_hour24() returns 10
c.get_hour12() returns 10
c.get_minute() returns 30
c.get_time_string(true) returns 10:30
c.get_time_string(false) returns 10:30am
Enter hours to advance: 
Enter minutes to advance: 
Advancing c by 1 hours and 16 minutes.
After advancing, the time is 11:46
